lib/block-supports/typography.php

```diff
@@ -403,11 +405,15 @@ function gutenberg_get_computed_fluid_typography_value( $args = array() ) {
 
 	// Build CSS rule.
 	// Borrowed from https://websemantics.uk/tools/responsive-font-calculator/.
-	$view_port_width_offset = round( $minimum_viewport_width['value'] / 100, 3 ) . $font_size_unit;
+	$view_port_width_offset = gutenberg_get_typography_value_and_unit( ( $minimum_viewport_width['value'] / 100 ) . $font_size_unit )['combined'];
```

Comment:
Just double-check my reading here: it's safe to access `['combined']` here without checking it's set, because it's guaranteed to be as `$minimum_viewport_width` is sure to be an actual / valid value because if it wasn't this function would have returned early?

Or to put it differently: at first I saw the call to `gutenberg_get_typography_value_and_unit` and thought it might be dangerous because the function can technically return `null`, but in practice here it looks like it never will / it'll always return a value with `combined` available 👍

Reply:
> because it's guaranteed to be as $minimum_viewport_width is sure to be an actual / valid value because if it wasn't this function would have returned early?

Exactly! But I've replace this with a regular string replace for another reason 😆 

It's a micro optimization: don't run the viewport offset value through `gutenberg_get_typography_value_and_unit()` as the values have already been checked. 

It avoids another regex parsing.